x-pack/test/functional/apps/monitoring/cluster/overview.js

```diff
@@ -159,5 +164,51 @@ export default function ({ getService, getPageObjects }) {
         expect(await overview.doesLsPanelExist()).to.be(false);
       });
     });
+
+    describe('Alerts', () => {
+      const { setup, tearDown } = getLifecycleMethods(getService, getPageObjects);
+
+      before(async () => {
+        await setup('x-pack/test/functional/es_archives/monitoring/singlecluster_green_gold', {
+          from: 'Aug 23, 2017 @ 21:29:35.267',
+          to: 'Aug 23, 2017 @ 21:47:25.556',
+        });
+      });
+
+      after(async () => {
+        await tearDown();
+
+        const apiResponse = await supertest.get('/api/alerts/_find?per_page=20');
+        const monitoringAlerts = apiResponse.body.data.filter(
+          ({ consumer }) => consumer === 'monitoring'
+        );
+
+        await Promise.all(
+          monitoringAlerts.map(async (alert) =>
+            supertest.delete(`/api/alerts/alert/${alert.id}`).set('kbn-xsrf', 'true').expect(204)
+          )
+        );
```

Comment:
Nit: Maybe it's worth creating a test helper for this (since it's used in two places), that would also make it clear that it's doing clean up of the created alerts

Reply:
I added it as a new service